include/dca/parallel/no_concurrency/serial_collective_sum.hpp

```diff
@@ -60,7 +54,7 @@ class SerialCollectiveSum {
   template <typename T>
   void leaveOneOutAvg(T&) const {}
   template <typename T>
-  void leaveOneOutSum(T&) const {}
+  void leaveOneOutSum(T&, bool = false) const {}
```

Comment:
Needs doc not everyone will understand this idiom

Reply:
Gave a commented out name in 7586540c6a2bd8e716e46595ff48c4b4f291fea3